yolox_torchscript.py

```diff
@@ -0,0 +1,13 @@
+import torch
+import torch.nn as nn
+from yolox.models.yolox import YOLOX
+
+
+class DeployModule(nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.yolox = YOLOX()
+
+    def forward(self, x: dict[str, dict[str, torch.Tensor]], targets=None):
```

Comment:
imo we can simplify to expect only tensor with image if any other pre / post processing is outside the graph